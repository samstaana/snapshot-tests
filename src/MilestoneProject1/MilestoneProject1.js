import React from "react";

function MilestoneProject1() {
    return (
        <div id="container">
            <div id="start">Start Quiz!</div>
            <div id="quiz" style="display: none">
                <div id="question"></div>
                <div id="qImg"></div>
                <div id="choices">
                    <div class = "choice" id = "A" onclick = "checkAnswer('A')"></div>
                    <div class = "choice" id = "B" onclick = "checkAnswer('B')"></div>
                    <div class = "choice" id = "C" onclick = "checkAnswer('C')"></div>
                    <div class = "choice" id = "D" onclick = "checkAnswer('D')"></div>
                </div>
                <div id="timer">
                    <div id="counter"></div>
                    <div id="btimeGauge"></div>
                    <div id="timeGauge"></div>
                </div>
            </div>
        </div>
    );
}

export default MilestoneProject1;