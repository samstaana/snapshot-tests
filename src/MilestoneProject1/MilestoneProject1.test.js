import MilestoneProject1 from './MilestoneProject1'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<MilestoneProject1/>).toJSON()
  expect(tree).toMatchSnapshot()
})