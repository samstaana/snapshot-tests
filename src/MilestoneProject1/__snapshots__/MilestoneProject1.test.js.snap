// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  id="container"
>
  <div
    id="start"
  >
    Start Quiz!
  </div>
  <div
    id="quiz"
    style="display: none"
  >
    <div
      id="question"
    />
    <div
      id="qImg"
    />
    <div
      id="choices"
    >
      <div
        class="choice"
        id="A"
        onclick="checkAnswer('A')"
      />
      <div
        class="choice"
        id="B"
        onclick="checkAnswer('B')"
      />
      <div
        class="choice"
        id="C"
        onclick="checkAnswer('C')"
      />
      <div
        class="choice"
        id="D"
        onclick="checkAnswer('D')"
      />
    </div>
    <div
      id="timer"
    >
      <div
        id="counter"
      />
      <div
        id="btimeGauge"
      />
      <div
        id="timeGauge"
      />
    </div>
  </div>
</div>
`;
